spec/models/framework/definition_spec.rb

```diff
@@ -4,89 +4,32 @@
   describe '.[]' do
     subject(:definition) { Framework::Definition[framework_short_name] }
 
-    context 'the framework is RM6060, because we are trialling FDL with this framework' do
-      let(:framework_short_name) { 'RM6060' }
-      it 'is a functioning framework' do
-        expect(definition.framework_short_name).to eql(framework_short_name)
-      end
+    context 'and it is loaded from the database' do
```

Comment:
The "and" at the start of the context is because this example was nested inside another context ("the framework is RM6060, because we are trialling FDL with this framework"). Now that this is no longer the case, the "and" doesn't read very well.